lang/en.json

```
@@ -218,6 +218,7 @@
 "DND5E.Charges": "Charges",
 "DND5E.ChatContextDamage": "Apply Damage",
 "DND5E.ChatContextHealing": "Apply Healing",
+"DND5E.ChatContextTempHP": "Apply Temp HP",
```

This should probably be
```suggestion
"DND5E.ChatContextTempHP": "Apply Temporary HP",
```

---

Agree, should be "Temporary"